woocommerce/woocommerce-gutenberg-products-block
# Product Gallery Pager: Remove the Pager markup if there's only one image

This PR prevents the **Pager** block from being outputted on the page if there's only **one** image in the Product Gallery.



## What

Fixes #10799 

## Why

The **Pager** block doesn't have a purpose if there's only one product image in the product gallery and there's nothing to cycle through. 

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. With a block theme, open `Appearance > Editor`.
2. Open `Templates > Single Product`.
3. Select `Convert to Blocks` if you are using the Classic Templates.
4. Add the `Product Gallery` block.
5. Ensure the `Pager` is enabled.
6. Open a product page on the frontend with multiple product images and verify the pager is displaying correctly.
7. Open a product page on the frontend with just one product image and using the `Inspector` verify the pager markup is not present.


## Screenshots or screencast



| Product with one image | Product with multiple images |
| ------ | ----- |
|<img width="1236" alt="Album_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/7cb143cc-8fde-437b-aa1a-f547e9cd2bac">|<img width="1197" alt="V-Neck_T-Shirt_–_newproductgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/73b34e33-e047-4180-8931-50fe919e384a">|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Product Gallery: Remove the Pager markup if a product has only one product image.
